modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/ComponentMessageProcessorObjectFactory.java

```diff
@@ -63,7 +63,8 @@ public P doGetObject() throws Exception {
       componentModel.getNestedComponents().stream()
           .filter(component -> component instanceof NestedChainModel)
           .findFirst()
-          .ifPresent(chain -> parameters.put(chain.getName(), new ProcessorChainValueResolver(nestedProcessors)));
+          .ifPresent(chain -> parameters.put(chain.getName(),
```

Comment:
superflous

Reply:
adding the `muleContext` as constructor parameter, that's the reason of the line break